pkg/network/infraconfigurators/BUILD.bazel

```diff
@@ -6,7 +6,6 @@ go_library(
         "bridge.go",
         "common.go",
         "generated_mock_common.go",
-        "macvtap.go",
```

Comment:
Thanks for making it obvious macvtap requires nothing on phase#1.

This commit alone is motivation enough for this PR.